scripts/content-repo-preview/build.ts

```diff
@@ -101,14 +96,8 @@ async function main() {
   /** Build */
   execFileSync('npm', ['run', 'build'], { stdio: 'inherit' })
 
-  // Copy the .next folder to the top-level so the site can be deployed and we can take advantage of the cache
-  // We are using cp here because the Vercel build is still running in the context of the nested website-preview directory,
-  // and so it needs the .next folder available to complete the deployment
-  execFileSync('cp', ['-R', '.next', '../.next'], { stdio: 'inherit' })
```

Comment:
We're pointing the `outputDirectory` vercel config option to `website-preview/.next` going forward, so we don't need to check for it at the top level like this.